src/doc/index.md

```diff
@@ -5,15 +5,14 @@ to jump to any particular section.
 
 # Getting Started
 
-If you haven't seen Rust at all yet, the first thing you should read is the [30
-minute intro](intro.html). It will give you an overview of the basic ideas of Rust
-at a high level.
-
-Once you know you really want to learn Rust, the next step is reading [The
-Rust Programming Language](book/index.html). It is a lengthy explanation of
-Rust, its syntax, and its concepts. Upon completing the book, you'll be an
-intermediate Rust developer, and will have a good grasp of the fundamental
-ideas behind Rust.
+If you haven't seen Rust at all yet, the first thing you should read is the
+introduction to [The Rust Programming Language](book/index.html). It'll give
+you a good idea of what Rust is like.
+
+The book provides a lengthy explanation of Rust, its syntax, and its
+concepts. Upon completing the book, you'll be an intermediate Rust
+developer, and will have a good grasp of the fundamental ideas behind
+Rust.
```

Comment:
to reduce awkward repetition: "behind Rust" -> "behind the language"

I'd go as far as removing the "you'll be an intermediate Rust developer"

I'm also wondering if "explanation of Rust" excludes Rust concepts, and I wonder if there's a need to mention that the book will also talk syntax (obvious). I'd trim things so that we'd have something like:

>If you haven't seen Rust at all yet, the first thing you should read is [the introduction](book/index.html). It'll give
you some idea of what Rust is like. The book will explain the language in much greater detail. You should have a good grasp of the fundamental ideas upon completing it.